### DISCUSSION TITLE
Change keybind?

### DISCUSSION THREAD

**Discussion by TheTripleJHD** [winstxnhdw/lc-hax#483] [Q&A]
Is there anyway to change the keybinds my keyboard does not have a pause button to turn off ESP.

**Comment by winstxnhdw** [comment]
See #407

**Reply by TheTripleJHD** [reply]
(() => Keyboard.current[Key.Digit8].wasPressedThisFrame, () => InputListener.OnPausePress?.Invoke()),

Just make sure its listening for the key I want and then pass the OnPausePress function that it was pressed?
Thank you for your help.

**Reply by winstxnhdw** [reply]
You only have to change Key.Pause to whatever key you want which seems like what you are doing.
